bin/install-prerelease.sh

```diff
@@ -13,13 +23,27 @@ fi
 
 source "$HOME/.nvm/nvm.sh"
 
-NVM_NODEJS_ORG_MIRROR="https://nodejs.org/download/rc"
-if ! nvm install "$PRERELEASE_NODE_VERSION" &> /dev/null; then
-  echo "RC installation failed. Trying Nightly mirror..."
+install_node_version() {
+  local PRERELEASE_NODE_VERSION=$1
+  echo "Installing Node.js prerelease version $PRERELEASE_NODE_VERSION..."
 
-  NVM_NODEJS_ORG_MIRROR="https://nodejs.org/download/nightly"
-  if ! nvm install "$PRERELEASE_NODE_VERSION" &> /dev/null; then
-    echo "Installation failed with both RC and Nightly mirrors."
-    exit 1
+  NVM_NODEJS_ORG_MIRROR="https://nodejs.org/download/rc"
+  export NVM_NODEJS_ORG_MIRROR
+
+  if ! nvm install "$PRERELEASE_NODE_VERSION"; then
+    echo "RC installation failed. Trying Nightly mirror..."
+    NVM_NODEJS_ORG_MIRROR="https://nodejs.org/download/nightly"
+    export NVM_NODEJS_ORG_MIRROR
+
+    if ! nvm install "$PRERELEASE_NODE_VERSION"; then
+      echo "Installation failed with both RC and Nightly mirrors."
+      echo "Trying to install configured version $CONFIGURED_PRERELEASE_NODE_VERSION..."
```

Comment:
What is the purpose of `CONFIGURED_PRERELEASE_NODE_VERSION`? What is its precedence? I believe we may not need it. 

Based on the code, it appears that even if `CONFIGURED_PRERELEASE_NODE_VERSION` is configured, the next major version takes precedence. Additionally, the nightly release for the upcoming version becomes available the day after the major release. Therefore, in my opinion, it seems unnecessary.

Reply:
If we don't need the configuration, it can be removed without harm!

Reply:
Yes, I agree. We can remove the config option and relay on the latest prerelease